tee-worker/ts-tests/integration-tests/di_evm_identity.test.ts

```diff
@@ -529,15 +469,24 @@ describe('Test Identity (evm direct invocation)', function () {
             requestIdentifier
         );
 
-        const res = await sendRequestFromTrustedCall(
-            context.tee,
-            context.api,
-            context.mrEnclave,
-            teeShieldingKey,
-            deactivateIdentityCall
-        );
+        const res = await sendRequestFromTrustedCall(context, teeShieldingKey, deactivateIdentityCall);
         assert.isTrue(res.do_watch.isFalse);
         assert.isTrue(res.status.asTrustedOperationStatus[0].isInvalid);
+        assertWorkerError(
```

Comment:
Can we standardize the assert functions, for example, by adding `export async function ...` and consistently using `await` when calling them? It's just for uniformity, without any specific implication. :p

Just feel that mixing "await" and not using it inconsistently can be a bit confusing

Reply:
I don't see a point in doing that really.
But I can add extra `async` and `await` keywords here and there but I would like to hear more voices on that, maybe @grumpygreenguy ?

Reply:
The main issue I see here is the risk of having an async assertion that we're not awaiting, e.g. 

```
    const events = await eventsPromise;
    assertFailedEvent(...); // oops, forgot await
```

If we wanted to standardize, I'd actually push for making all assertions synchronous :P
It'll take some dependency injection to get it to work tho

Reply:
Thanks for your voice @grumpygreenguy .